test/e2e/testnet/testnet.go

```
@@ -384,22 +392,24 @@
 		}
-		for i := 0; i < 10; i++ {
+		for i := 0; i < 20; i++ {
 			resp, err := client.Status(ctx)
 			if err == nil {
-				if resp.SyncInfo.LatestBlockHeight > 0 {
-					log.Info().Int("attempts", i).Str("name", node.Name).Msg(
+				if resp != nil && resp.SyncInfo.LatestBlockHeight > 0 {
+					log.Info().Int("attempts", i).Str("name", node.Name).Int64("latest_block_height", resp.SyncInfo.LatestBlockHeight).Msg(
 						"node has synced")
 					break
+				} else {
```

golangci-lint (1.62.2)

[warning] superfluous-else: if block ends with a break statement, so drop this else and outdent its block

(revive)